ExceptionHandling/ExceptionHandling/AppInteraction/ArrayApplication.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ExceptionHandling.Exceptions;
+using ExceptionHandling.IOManager;
+
+namespace ExceptionHandling.AppInteraction {
+    internal class ArrayApplication {
+        static int[] numbers = new int[] { 2, 3, 5, 7, 11, 13, 17, 19 };
```

Comment:
I think we have not modified the array anywhere. Moreover, let's simplify the expression like below

```suggestion
        private static readonly int[] numbers = [2, 3, 5, 7, 11, 13, 17, 19];
```

Reply:
Resolved.